### DISCUSSION TITLE
How to use NuxtImg as background img with tailwind class bg-[url(xxx)] ?

### DISCUSSION THREAD

**Discussion by GuillaumeDgr** [nuxt/image#1369] [Q&A]
How to use NuxtImg as background img with tailwind class bg-[url(xxx)] ?

**Comment by TechAkayy** [comment]
Here is an example - https://github.com/Pinegrow/happy-paws-with-nuxt-tailwindcss/blob/main/app/components/BackgroundImageWrapper.vue. This component uses a local composable (https://github.com/Pinegrow/happy-paws-with-nuxt-tailwindcss/blob/main/app/composables/optimize-image.ts) that uses the useImage composable to achieve this.
